src/projections.jl

```diff
@@ -84,30 +84,59 @@ lat2colat, colat2lat
 ################################################################################
 
 """
-    function equiprojinv(x, y)
+    equiproj(lat, lon)
+
+Equirectangular projection. Given the latitude `lat` (in radians) and the
+longitude (in radians), return a tuple (Bool, Number, Number) where the
+first Boolean is a flag telling if the point falls within the projection (true)
+or not (false), and the two numbers are the x and y coordinates of the point
+on the projection plane (both are in the range [−1, 1]).
+"""
+function equiproj(lat, lon)
+    # The weird usage of mod2pi(x + π) - π allows to have angles in the range
+    # [-π, +π]
+    x, y = ((mod2pi(lon + π) - π) / π, 2 * (mod2pi(lat + π) - π) / π)
+    (true, x, y)
+end
+
+"""
+    equiprojinv(x, y)
 
 Inverse equirectangular projection. Given a point (x, y)
 on the plane [-1, 1] × [-1, 1], return a tuple (Bool, Number, Number)
 where the first Boolean is a flag telling if the point falls
 within the projection (true) or not (false), and the two numbers
-are the latitude and colatitude in radians.
+are the latitude and longitude in radians.
 """
-function equiprojinv(x, y; kwargs...)
+function equiprojinv(x, y)
     ((-1 ≤ x ≤ 1) && (-1 ≤ y ≤ 1)) || return (false, 0, 0)
      
     (true, π / 2 * y, π * x)
 end
 
 """
-    function mollweideprojinv(x, y)
+    mollweideproj(lat, lon)
+
+Mollweide projection. Given the latitude `lat` (in radians) and the
+longitude (in radians), return a tuple (Bool, Number, Number) where the
+first Boolean is a flag telling if the point falls within the projection (true)
+or not (false), and the two numbers are the x and y coordinates of the point
+on the projection plane (both are in the range [−1, 1]).
+"""
+function mollweideproj(lat, lon)
+    (true, sin(lat), 2 / π * lon * cos(lat))
```

Comment:
You could use `sincos(lat)` to compute the sine and the cosine at the same time (you already did so in other places)

Reply:
Sure, I have changed the code.